### ISSUE TITLE
String can't be coerced into Float

### ISSUE THREAD

**Issue by ste26054** [imaginary-cloud/redmine_evm#57]
Hello,

I am having troubles with the plugin when a baseline is set:

Message:
String can't be coerced into Float

Params:
action: show
controller: baselines
id: 1

Backtrace:
/usr/share/redmine/plugins/redmine_evm/app/models/baseline.rb:105:in `-'
/usr/share/redmine/plugins/redmine_evm/app/models/baseline.rb:105:in`cost_variance'
/usr/share/redmine/plugins/redmine_evm/app/helpers/baselines_helper.rb:24:in `label_schedule_performance_percentage'
/usr/share/redmine/plugins/redmine_evm/app/views/baselines/show.html.erb:61:in`block in _plugins_redmine_evm_app_views_baselines_show_html_erb___1336144061593088855_69974117890080'

Here is my environment:
  Redmine version         2.5.1.stable
  Ruby version               1.9.3-p484 (2013-11-22) [x86_64-linux]
  Rails version                3.2.16
  Environment                 production
  Database adapter         MySQL

Is this an issue with rails ?

Thanks,
Stephane

**Comment by ste26054** [comment]
Hmm

I did not notice the tiny rails version... (3.2.16 vs 3.2.17)

I am going to update and see if this changes anything.

**Comment by MarioVillani** [comment]
I'm running ruby 1.9.3p484 (2013-11-22 revision 43786) [x86_64-linux]
and Rails 3.2.16 like you and i have the same issue.

I've done some debugging and i've found what the bug is related to:

If you look at /redmine_evm/app/models/baseline.rb you'll notice this code from line 105

```
  #Cost Performance Index (CPI)
  def cost_performance_index
    project.actual_cost(self) != 0 ? project.earned_value(self).to_f / project.actual_cost(self) : 0
  end

  #Schedule Variance (SV)
  def schedule_variance
    project.earned_value(self) - self.planned_value
  end

  #Cost Variance (CV)
  def cost_variance
    project.earned_value(self) - project.actual_cost(self)
  end
```

Newer Ruby and Rails versions do not manage to convert this three instruction from Float to String. I'll try to fix that.

**Comment by MarioVillani** [comment]
I've installed your forked version and the problem is gone. 
Your fix should be included in the master branch! Thank you very much.

**Comment by Sousa5** [comment]
Hi @ste26054 and @MarioVillani 

I've released a new version (1.0.1) with this fix.

Thanks for your help.
